libraries/disp/viewers/averagelayoutview.h

```diff
@@ -239,8 +256,12 @@ class DISPSHARED_EXPORT AverageLayoutView : public AbstractView
     QSharedPointer<DISPLIB::EvokedSetModel>                     m_pEvokedSetModel;          /**< The data model */
     QSharedPointer<DISPLIB::ChannelInfoModel>                   m_pChannelInfoModel;        /**< Channel info model. */
 
+    QSharedPointer<FIFFLIB::FiffInfo>                           m_pFiffInfo;                /**< FiffInfo for currently loaded file */
+
     QPointer<QGraphicsView>                                     m_pAverageLayoutView;       /**< View for 2D average layout scene */
 
+    SelectionItem*                                              m_pSelItem;
```

Comment:
are you making sure that these are either passed to a parent/child relationship or deleted manually?

Reply:
Removed as a member variable